ExtractUserDataActivity/__tests__/handler.test.ts

```diff
@@ -255,7 +291,8 @@ describe("createExtractUserDataActivityHandler", () => {
       notificationStatusModel: notificationStatusModelMock,
       profileModel: profileModelMock,
       userDataBlobService: blobServiceMock,
-      userDataContainerName: aUserDataContainerName
+      userDataContainerName: aUserDataContainerName,
+      servicePreferencesModel
```

Comment:
I'd maintain coherence keeping or `servicePreferencesModel: servicePreferencesModel` or `servicePreferencesModel` alone. Also mocks have usually a "Mock" suffix.

Reply:
done in https://github.com/pagopa/io-functions-admin/pull/191/commits/14030001890cd960f5309d3f6bac471151235df0